Gui/opensim/installer_files/Linux/INSTALL

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+function err_unsupported_env {
+    printf "\r  [\033[0;31mERROR:\033[0m] Your distro/release is not officially supported\n"
+    printf "  Ubuntu 18.04 LTS is currently the only Linux distribution that is officially \
+supported, however the OpenSim GUI will likely run on other distributions and/or \
```

Comment:
I would prefer that we do not claim that we officially support any Linux version.

Can you change "that is officially supported" to "that has been tested"

Reply:
Oops! Must have missed that when I was changing everything else to "beta" language.